src/gui/window_file_list.cpp

```diff
@@ -20,8 +19,10 @@
     #include "bsod.h"
 #endif
 
+char *window_file_list_t::root = nullptr;
+
 bool window_file_list_t::IsPathRoot(const char *path) {
-    return (path[0] == 0 || strcmp(path, "/") == 0);
+    return (path[0] == 0 || (root && strcmp(path, root) == 0));
```

Comment:
If the path is empty are equals to e.g. "/usb" do we really want to consider it as the same?